WooCommerce.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,196 +1,204 @@
 {
-  "object": {
```

Comment:
I was actually expecting this file to disappear, given that there are no longer any packages defined via the "Package Dependencies" in the project.

I think this might be due to the workspace setup being in place because of CocoaPods. I expect, hope, once WooCommerce is the only project in the workspace (and eventually we'll remove the workspace altogether in favor of the project only) this will disappear and `Modules/Package.resolved` will be the source of truth. For the time being, I think we can cope with this duplication.

As far as I've seen, this is the one that Xcode updates when resolving packages.